src/euphorie/client/model.py

```diff
@@ -799,6 +799,20 @@ def refresh_survey(self, survey=None):
                     item.title = zodb_item.title
         self.refreshed = datetime.datetime.now()
 
+    def update_measure_types(self, survey):
+        in_place_standard_measures = (
+            Session.query(Risk, ActionPlan)
+            .filter(Risk.id == ActionPlan.risk_id)
+            .filter(Risk.session_id == self.id)
+            .filter(ActionPlan.plan_type == "in_place_standard")
+            .all()
+        )
+        for risk, measure in in_place_standard_measures:
+            risk_zodb = survey.restrictedTraverse(risk.zodb_path.split("/"))
+            solution_ids = [sol.id for sol in risk_zodb._solutions]
+            if measure.solution_id not in solution_ids:
```

Comment:
```suggestion
            if measure.solution_id not in solution_ids:
                # Measure not available because it was deleted in CMS.
                # Update type to ``in_place_custom`` do make them show up in existing sessions.
```

Reply:
I'd add a little more context information - but I'm not sure if my wording is correct...